### DISCUSSION TITLE
Add the osu!stream intro and main menu music

### DISCUSSION THREAD

**Discussion by LeNitrous** [ppy/osu#25596] [Ideas]
Including this part of osu! history would be great for those wanting nostalgia back when they were first introduced to the game through osu!stream. This proof of concept attempts to faithfully recreate the main menu experience as much as possible using the available source code and assets via https://github.com/ppy/osu-stream.

https://github.com/ppy/osu/assets/20495991/40bcd385-fb15-4d92-98ce-309fe61ca5ef

The PoC uses main menu 2.0 but adding support for the original main menu with a few tweaks.

**Comment by peppy** [comment]
I dunno about this one since it's a different game, but I guess we'll see.
